ports/net/macathena-athrun/Portfile

```diff
@@ -1,17 +1,35 @@
 # $Id$
 PortSystem		1.0
+PortGroup		github 1.0
+PortGroup		python 1.0
 
 name			macathena-athrun
-version			20060725
-distname		athrun-${version}
+github.setup		macathena athrun 3c9c5b72414454df3c4faad6d92929c5b6e0dc2b
+version			10.3.2.1
 categories		net
 maintainers		mit.edu:sipb-macathena
+python.default_version	38
 platforms		darwin
-master_sites	http://macathena.mit.edu/dist/
-checksums		md5 0c96055a5fbf52736f2a85103f207fd3 \
-				sha1 5554253d27513c84170b763cf73f2d13343a9f32 \
-				rmd160 25e39e7e51e63a14feb5c78eeea421e66419603d
+checksums           rmd160  72198fa0fcad450c28b9f002ba95a2a1d929e360 \
+                    sha256  0e041cf633e043a12cfc0a71bdb20ce99b933cadd9641958c939ac9260197bd6 \
+                    size    3529
 
-use_autoconf	yes
+post-destroot {
```

Comment:
I think you need to protect this with something like `if {${name} == ${subport}}` so it doesn't try to install these files in macathena-gathrun as well. Currently the `post-destroot` block is being run by the `gathrun` subport as well, so the resulting `gathrun` port conflicts with the `athrun` port.